experimental/benchmarks/sdxl/benchmark_sdxl_rocm.py

```diff
@@ -0,0 +1,407 @@
+# Copyright 2024 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+import os
+from collections import namedtuple
+import logging
+from typing import Sequence
+import subprocess
+import json
+from pathlib import Path
+import tabulate
+from pytest_check import check
+
+benchmark_dir = os.path.dirname(os.path.realpath(__file__))
+artifacts_dir = os.getenv("IREE_TEST_FILES", default=Path.cwd()) + "/artifacts"
+artifacts_dir = Path(os.path.expanduser(artifacts_dir)).resolve()
+prompt_encoder_dir = f"{artifacts_dir}/sdxl_clip"
+scheduled_unet_dir = f"{artifacts_dir}/sdxl_unet"
+vae_decode_dir = f"{artifacts_dir}/sdxl_vae"
+
+
+def run_iree_command(args: Sequence[str] = ()):
+    command = "Exec:", " ".join(args)
+    logging.getLogger().info(command)
+    proc = subprocess.run(
+        args, stdout=subprocess.PIPE, stderr=subprocess.PIPE, check=False
+    )
+    (
+        stdout_v,
+        stderr_v,
+    ) = (
+        proc.stdout,
+        proc.stderr,
+    )
+    return_code = proc.returncode
+    if return_code == 0:
+        return 0, proc.stdout
+    logging.getLogger().info(f"Command failed with error: {proc.stderr}")
+    return 1, proc.stdout
```

Comment:
This has the same bug from https://github.com/nod-ai/SHARK-TestSuite/issues/286. Once landed we can iterate on a fix in this repo.